File: server/Models/RecipeFan.cs
namespace allspice.Models;

public class RecipeFan
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Title { get; set; }
    public string Instructions { get; set; }
    public string Img { get; set; }
    public string Category { get; set; }
    public string CreatorId { get; set; }
    public Profile Creator { get; set; }
    public Favorite Favorite { get; set; }
    public Profile Fan { get; set; }
}

File: client/src/models/RecipeFan.js

export class RecipeFan{
    constructor(data){
        this.id = data.id
        this.creatorId = data.creatorId
        this.createdAt = data.createdAt
        this.updatedAt = data.updatedAt
        this.title = data.title
        this.instructions = data.instructions
        this.img = data.img
        this.category = data.category
        this.creator = data.creator
        this.favorite = data.favorite
        this.fan = data.fan
    }
}